float64[7] tau_d
# float64[7] tau_d_fli
float64[7] tau_J
#
# float64[7] q
# float64[7] dq
# float64[7] q_d
# float64[7] dq_d
# float64[7] qError
# float64[7] dqError

# float64[6] X
# float64[6] dX
# float64[6] X_d
# float64[6] dX_d
# float64[6] Xerror
# float64[6] dXerror

float64[3] pos
float64[3] dpos
# float64[3] ddpos
float64[3] pos_a
float64[3] dpos_a
# float64[3] ddpos_a
float64[3] pos_d
float64[3] dpos_d

float64[3] ori
float64[3] ori_d
float64[3] pos_error
float64[3] ori_error
# float64[3] pos_ACerror
# float64[3] dpos_ACerror
# float64[3] ddpos_ACerror
float64[6] K
float64[6] D

float64[6] F_sensor
float64[6] F_sensor_fil
float64[6] F_ext0
float64[6] F_extK
float64[6] F_c
float64 depthDistance
float64 depthDistance_filter
float64 VirtualForce

float64 error_z
float64 time
float64 run_time
float64 x_ba
float64 error_z_avg


# float64[7] D
# float64[3] K
